BootloaderCorePkg/Stage1A/Ia32/Vtf0/Main.asm

```diff
@@ -34,5 +34,7 @@ BITS    32
     ;
     mov     eax, 0FFFFFFFCh    
     mov     eax, dword [eax]
-    mov     esi, dword [eax]                  
+    mov     esi, dword [eax]               
+    ; Restore the BIST value to EAX register
+    movd    eax, mm0   

```

Comment:
Looks good to me. Can you add more thing to handle this BIST value in Stage1A? The eax is overwritten by rstsc in _ModuleEntryPoint of Stage1A. It should be saved in STAGE1A_ASM_HOB for next step to handle BIST failure.

Reply:
@Aiden,  for the BIST handling,  how about handing it in a separate patch ?

Reply:
Not a problem with a separate patch.